### PR TITLE
Add cli flag to enable monitoring

### PR DESCRIPTION
**Pull request by palango**
Some progress towards #3273

### CODE REVIEW

**Comment by LefterisJP** [comment]
Reviewing this one

File: raiden/ui/cli.py [hunk 1]
```diff
@@ -218,6 +218,11 @@ def options(func):
                 type=int,
                 show_default=True,
             ),
+            option(
+                '--enable-monitoring',
+                help='enabled the broadcasting of balance proofs to the monitoring services',
```
Comment (LefterisJP) [inline raiden/ui/cli.py]: typo: Should be: "enable" I guess?

File: raiden/raiden_event_handler.py [hunk 2]
```diff
@@ -540,3 +540,24 @@ def handle_contract_send_channelsettle(
...
+
+
+class RaidenMonitoringEventHandler(RaidenEventHandler):
+    # pylint: disable=no-self-use
+
+    def on_raiden_event(self, raiden: RaidenService, event: Event):
+        if type(event) == EventPaymentReceivedSuccess:
+            self.handle_received_payment(raiden, event)
+
+        # Call base class to trigger other side-effects
```
Comment (LefterisJP) [inline raiden/raiden_event_handler.py]: nitpick: I would phrase it as: "Call base class for processing of all other event types"

Comment (palango) [reply raiden/raiden_event_handler.py]: It might be the same event again as well, this is just a decorator.

Comment (LefterisJP) [reply raiden/raiden_event_handler.py]: Hmm indeed. Disregard my comment. Was thinking of something completely different.

**Pull request merged**